src/auth/jwt-auth.guard.ts

```diff
@@ -0,0 +1,4 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+@Injectable()
+export class JwtAuthGuard extends AuthGuard('jwt') {}
```

Comment:
PrimathonAICodeReviewer

Consider adding a constructor to initialize any necessary properties in the `JwtAuthGuard` class.

```diff
+     constructor() {
+         super();
+     }
```


<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->